lgc/builder/BuilderBase.cpp

```diff
@@ -28,14 +28,43 @@
  * @brief LLPC source file: implementation of BuilderBase
  ***********************************************************************************************************************
  */
+
 #include "lgc/util/BuilderBase.h"
+#include "lgc/LgcDialect.h"
+#include "lgc/state/IntrinsDefs.h"
 #include "llvm/IR/InlineAsm.h"
 #include "llvm/IR/IntrinsicInst.h"
 #include "llvm/IR/IntrinsicsAMDGPU.h"
 
 using namespace lgc;
 using namespace llvm;
 
+// =====================================================================================================================
+// Return the i64 difference between two pointers, dividing out the size of the pointed-to objects.
+// For buffer fat pointers, delays the translation to patch phase.
+//
+// @param ty : Element type of the pointers.
+// @param lhs : Left hand side of the subtraction.
+// @param rhs : Reft hand side of the subtraction.
+// @param instName : Name to give instruction(s)
```

Comment:
I think for newly added function, we could start to add "@return : " comments for those have return types. This was once requested by Jakub as a coding style.